strings/traumas.json

```diff
@@ -53,14 +55,14 @@
         "@pick(semicolon)I'VE GOT BALLS OF STEEL",
         "NO I'M ONNA KILL YOU MOTHERFUCKER OLD STYLE",
         "i will snatch erry motherfucker birthday",
-        "SHEDOWLINGS G0T NO ABS!",
+        "@pick(semicolon)the ai and borgs are mettacomming I think",
```

Comment:
:eyes:

Reply:
That brain damage line is a work of art. It's a shame it's too OOC.